.hlint.template.yaml

```diff
@@ -0,0 +1,70 @@
+# HLint configuration file
+# https://github.com/ndmitchell/hlint
+##########################
+
+# This file contains a template configuration file, which is typically
+# placed as .hlint.yaml in the root of your project
```

Comment:
Why is this template file included in the PR?

Reply:
Removed.  Thanks!